p5js/BodyPix/BodyPix_Webcam_Parts/sketch.js

```diff
@@ -0,0 +1,38 @@
+
+console.log('bodyPix', ml5.bodyPix);
+
+let bodypix;
+let video;
+let segmentation;
+let img;
+function setup(){
+    createCanvas(320, 240);
+
+    // load up your video
+    video = createCapture(VIDEO);
+    video.size(width, height);
+    // video.hide(); // Hide the video element, and just show the canvas
+    bodypix = ml5.bodyPix(video, modelReady)
```

Comment:
seems a little strange that the `video` is passed in here and also `segmentWithParts()`?

Reply:
Also, this raises my usual question as to whether we should have:

```javascript
ml5.uNet('face');
ml5.bodyPix();
```

or

```javascript
ml5.segementation('uNet-Face');
ml5.segmentation('BodyPix');
```

Reply:
@shiffman - oOoOoH. I do like the latter:

`ml5.segmentation(<specified model name>)`

It would be similar to the way `imageClassification()` works. Though this might be a bigger restructuring?